src/pages/input-range.ts

```diff
@@ -123,10 +123,39 @@ function setPreview(values: Values): void {
   );
 }
 
+function areInputsValid() {
+  const getTrackColor = getColorInput1(attribute);
+  const getThumbColor = getColorInput2(attribute);
+
+  const areColorInputsValid = getTrackColor.value && getThumbColor.value;
+
+  const trackCheckBox = getCheckbox(`${attribute}-track`);
+  const getTrackRadius = getRadiusInput(`${attribute}-track`);
+
+  const isTrackRadiusInputValid = trackCheckBox.checked
+    ? getTrackRadius.value !== ''
+    : true;
+
+  const thumbCheckBox = getCheckbox(`${attribute}-thumb`);
+  const getThumbRadius = getRadiusInput(`${attribute}-thumb`);
+
+  const isThumbRadiusInputValid = thumbCheckBox.checked
+    ? getThumbRadius.value !== ''
+    : true;
+
```

Comment:
your issue isn't to check if their valid or not, it's to check if they exist, not part of your issue, remove it

Reply:
The function is checking if the input exists. Maybe I have chosen wrong names for the functions and the variables😅. I will make the desired changes.

Reply:
i know what the function does, my point is that isn't the job of your issue, you are going overboard which is something i don't accept in PRs

Reply:
I am a bit confused. Suppose the user has checked 'include thumb radius' but has not entered any value for the same. In such a case there may be two way of handling it. We can include a default value for the border radius in the css code or we may show a notification that 'some of the input values may be missing'. Which one would you prefer?

Reply:
i accept your change on the notification, i'm saying i do not accept your change on checking if they are valid, from the other things i see you just check if it has a value or not and leave it at that

Reply:
I believe that is the exact thing I am trying to achieve using this function. If the function is returning a true value we are proceeding with the copying of the css code else we are just displaying a missing value notification.

Linked review : https://github.com/Dun-sin/Code-Magic/pull/497#discussion_r1360952628

Reply:
then you need to change the name of the function to properly reflex the change, I assumed it wasn't needed because you did the same thing for the grid generator -> https://github.com/Dun-sin/Code-Magic/pull/497/files#r1361175725

Reply:
Okay, I will commit the necessary changes.